sass/components/variables.scss

```diff
@@ -0,0 +1,5 @@
+$default-page-content-width: 100%;
```

Comment:
it seems we were missed alot of other variables. all variabled of the app should be defined in this file